e2e_test/source_inline/kafka/consumer_group.slt

```diff
@@ -0,0 +1,101 @@
+# Note: control substitution on will force us to use "\\n" instead of "\n" in commands
+control substitution on
+
+# Note either `./risedev rpk` or `rpk` is ok here.
+# risedev-env contains env var RPK_BROKERS, which is read by rpk
+system ok
+rpk topic create test_consumer_group -p 3
+
+system ok
+cat <<EOF | rpk topic produce test_consumer_group -f "%p %v\\n" -p 0
+0 {"x":"a"}
+1 {"x":"b"}
+2 {"x":"c"}
+EOF
+
+statement ok
+CREATE SOURCE s(x varchar)
+WITH(
+    ${RISEDEV_KAFKA_WITH_OPTIONS_COMMON},
+    topic = 'test_consumer_group',
+    scan.startup.mode = 'earliest',
+) FORMAT PLAIN ENCODE JSON;
+
+statement ok
+CREATE MATERIALIZED VIEW mv AS SELECT * from s;
+
+query ?
+SELECT * FROM s order by x;
+----
+a
+b
+c
+
+sleep 2s
+
+query ?
+SELECT * FROM mv order by x;
+----
+a
+b
+c
+
+
+# There are 4 consumer groups, 1 for batch query (not listed below), 3 for MV.
+# All of them are "Empty" state with 0 members, because we manually `assign` partitions to them.
+# At the begginning, the MV's consumer group will not occur. They will be created after committing offset to Kafka.
+# (enable.auto.commit defaults to true, and auto.commit.interval.ms defaults to 5s)
+sleep 5s
+
+system ok
+./e2e_test/source_inline/kafka/consumer_group.mjs --mv mv list-members
+----
+0,0,0
+
+
+# The lag for batch query's group is 0, and each MV parition's group is 2 (1 of 3 consumed).
+system ok
+./e2e_test/source_inline/kafka/consumer_group.mjs --mv mv list-lags
```

Comment:
So this requires that current working directory is at the project root. Can we organize these utilities together and add them to the `PATH` somehow? 😄

Reply:
> So this requires that current working directory is at the project root. Can we organize these utilities together and add them to the PATH somehow? 😄

I also thought about this, but not very sure. It can indeed make the script look nicer. (There's also another benefit: we can make the scripts a "project", and do more powerful stuff like adding dependencies to it.)

1. `risedev` always runs in the project root. We assume the slt is run by `risedev slt`, not only for the path issue, but also env vars, so even if we changed it, we still need to use `risedev slt`.
2. As @stdrc mentioned https://github.com/risingwavelabs/risingwave/issues/12451#issuecomment-2055611502, (also like DocSlt) maybe orginaze everything together is clearer. Adding more indirection also increases burden to find the scripts. (first indirection is put command in script, second indirection is put the script somewhere else and in PATH). Although running the tests doesn't require multiple steps now, but maybe not good for reading/writing.

So I'm a little hesitant about this.

Reply:
I guess: if the script is ad-hoc (only used for one test), it's better to put next to the test file. If it is general (and as we have more such scripts), it's better to put together. It's kind of like "built-in" commands for the test driver. 😄

For this PR, I prefer not to change it as the benefits haven't shown.